framework/src/main/java/org/tron/core/consensus/BlockHandleImpl.java

```diff
@@ -54,9 +60,15 @@ public BlockCapsule produce(Miner miner, long blockTime, long timeout) {
       manager.pushBlock(blockCapsule);
       tronNetService.broadcast(blockMessage);
     } catch (Exception e) {
+      monitorMetric.getMeter(MonitorMetric.NODE_STATUS)
```

Comment:
NODE_STATUS Why is it set here ？ and doesn't need to be defined as metrics

Reply:
check node status, if there is any expection during a period, then set node status to inactive.